<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyMaterials</class>
 <widget class="QDialog" name="ModifyMaterials">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#1d1d1d;
}

QDialog{
background-color:#1d1d1d;
}

QFileDialog{
background-color: #1d1d1d;
}

QInputDialog{
background-color: #1d1d1d;
}

QWidget{
background-color: #1d1d1d;
}

QMenuBar{
background-color:#1d1d1d;
padding:5px;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QMenuBar::item{
background-color:#1d1d1d;
color:#fff;
padding:5px;
}

QMenu{
color:#fff;
padding:0;
}

QMenu::item:selected{
color:#fff;
background-color:#00aba9;
}

QSplitter::handle{
background-color: #1d1d1d;
}

QTableWidget{
background-color:#3d3d3d;
color:#fff;
border-radius: 4px;
}

QHeaderView::section{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 100, 150, 255), stop:1 rgba(36, 100, 150, 255));
border:none;
border-top-style:solid;
border-width:1px;
border-top-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 100, 150, 255), stop:1 rgba(36, 100, 150, 255));
color:#fff;

}
QHeaderView{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 100, 150, 255), stop:1 rgba(36, 100, 150, 255));

border:none;
border-top-style:solid;
border-width:1px;
border-top-color:#146496;
color:#fff;
	font: 75 12pt &quot;Calibri&quot;;
}

QTableCornerButton::section{
border:none;
background-color:#146496;
}

QListWidget{
background-color:#3d3d3d;
color:#fff;
border-radius: 8px;
}

QMenu{
background-color:#3d3d3d;
}

QStatusBar{
background-color:#7e3878;
color:#fff;
}

QPushButton{
min-width: 80px;
min-height: 30px;

border-style:solid;

background-color:#3d3d3d;
color:#fff;
border-radius:7px;
}

QPushButton:hover{
color:#ccc;
	background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(45, 45, 45, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
	border-color:#2d89ef;
border-width:2px;
}

QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(29, 29, 29, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
}


QTabWidget::tab{
background-color:#3d3d3d;
}

QLineEdit{
border-radius:0;
color: #fff;
}

QProgressBar{
border-radius:0;
text-align:center;
color:#fff;
background-color:transparent;
border: 2px solid #e3a21a;
border-radius:7px;
	font: 75 12pt &quot;Open Sans&quot;;

}

QProgressBar::chunk{
background-color:#2d89ef;
width:20px;
}

QComboBox{
background-color: #3d3d3d;
color: #fff;
}

QComboBox QAbstractItemView{
background-color: #3d3d3d;
color: #fff;
}

QTextBrowser{
background-color: #3d3d3d;
color: #fff;
border-radius: 8px;
}

QSpinBox{
background-color: #3d3d3d;
color: #fff;
font: 16pt
}

QLabel{
color: #fff;
}

QCheckBox{
color: #fff;
}</string>
  </property>
  <widget class="QLabel" name="Material_Icon">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>10</y>
     <width>85</width>
     <height>85</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="materialName">
      <property name="text">
       <string>Material</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>How many do you have?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="howManyMaterials"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="confirmChanges">
          <property name="text">
           <string>Confirm Changes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="discardChanges">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Discard Changes</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
